src/input/cmd.rs

```diff
@@ -910,3 +916,288 @@ fn cmd_write(args: &[u8], vga_index: &mut isize) {
     }
 }
 
+fn cmd_dd(args: &[u8], vga_index: &mut isize) {
+    if args.len() == 0 {
+        println!("Usage: dd if=<input> of=<output> [bs=<blocksize>] [count=<blocks>] [skip=<blocks>] [seek=<blocks>] [status=<level>]");
+        println!("       dd format [fs=<filesystem>] [drive=<device>]");
+        println!("Examples:");
+        println!("  dd if=file1.txt of=file2.txt bs=256 count=4    - copy with custom block size and count");
+        println!("  dd if=file1.txt of=file2.txt skip=2 seek=1     - copy with input/output offsets");
+        println!("  dd format fs=fat12                             - format with FAT12 filesystem");
+        println!("  dd format fs=raw                               - raw format (zero-fill)");
+        println!("  dd status=progress                             - show transfer progress");
```

Comment:
help strings also need to be updated to the syntax without dd format
status=progress is meaningless without any output location